packages/number-field/README.md

```diff
@@ -96,7 +119,7 @@ The following example uses the `signDisplay` option to include the plus sign for
 <sp-number-field
     id="decimals"
     value="0"
-    style="width: 100px"
+    style="width: 200px"
```

Comment:
Did we want to make this 400px like several of the others in this formatting section?

Reply:
I switched them all back to 200px. Thanks!